src/main/scala/edu/ie3/osmogrid/lv/LvCoordinator.scala

```diff
@@ -65,11 +68,18 @@ object LvCoordinator {
           val lvRegionCoordinatorProxy =
             ctx.spawn(lvRegionCoordinatorPool, "LvRegionCoordinatorPool")
 
-          replyTo ! RepLvGrids(Vector.empty[SubGridContainer])
-          Behaviors.stopped
+          replyTo ! RepLvGrids(Seq.empty[SubGridContainer])
+          Behaviors.stopped { () => cleanUp() }
         case unsupported =>
           ctx.log.error(s"Received unsupported message: $unsupported")
-          Behaviors.stopped
+          Behaviors.stopped { () => cleanUp() }
       }
-  }
+    }
+    .receiveSignal { case (ctx, PostStop) =>
+      ctx.log.info("Got terminated by ActorSystem.")
+      cleanUp()
+      Behaviors.same
+    }
+
+  private def cleanUp(): Unit = ???
```

Comment:
```suggestion
  private def cleanUp(): () => Unit = ???
```

Reply:
Don't see the benefit here of a function?!